client/tailscale/localclient.go

```diff
@@ -859,6 +859,34 @@ func (lc *LocalClient) NetworkLockSign(ctx context.Context, nodeKey key.NodePubl
 	return nil
 }
 
+func (lc *LocalClient) AddServeConfig(ctx context.Context, config ipn.ServeConfig) (*ipn.ServeConfigView, error) {
```

Comment:
Don't use pointers to reviews.

Here I'd just return the non-view and let the caller own the memory. The caller can make their own view if they want to.

Also: exported, so should have docs. But we can do that later.

I'd name this "SetServeConfig", as it blows away the old one, right? It doesn't add anything to the existing one.